src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
This rule will not work as expected. It will always apply the active star style to the first four stars, regardless of the rating. A better approach would be to control the number of active stars by adding an additional class to the star elements in your HTML, and then styling the active stars based on that class. For example, you could use classes like 'stars--1', 'stars--2', etc., where the number indicates the number of active stars.